sqls/issue_pr-resolve_til1stresponse-average_period/post-processor.js

```diff
@@ -0,0 +1,9 @@
+module.exports = async function(data) {
+  if (!data.count || Number.isNaN(parseInt(data.count))) {
+    throw new Error('Invalid data');
+  }
+  let ret = '| resolve_period_avg | respond_period_avg | resolve_period_median | respond_period_median | count |\n';
```

Comment:
You should also select `repo_name` out and add it into the output table.

Reply:
What if the repo_name in the manifest json is different from the repo_name queried from clickhouse? Should I return the any of those, or just prune the manifest?

Reply:
You can see how others do. Actually we only use repo id in manifest and get repo name by `anyHeavy` to get the most frequently used repo name in 2020. The repo names in manifest are just used for repo id double check.

Reply:
I am still confused. It's about the gather/pooling functions (average & median). I think it will return a single row which is covering the gathered massages of the input list(config.repos), rather than return lists grouped by the repos; therefore, I doubt it is neccessary to return a repo name in this case...

Reply:
Well, I see. I misunderstand the purpose of this issue...

Reply:
And actually this is a case study component, you should put them into case study folder.

Reply:
I've moved those new files into case study folder.